config/initializers/call_session_store.rb

```diff
@@ -0,0 +1,13 @@
+require 'connection_pool'
+
+DEFAULT_CALL_SESSION_REDIS_POOL_SIZE = 12
+DEFAULT_CALL_SESSION_REDIS_TIMEOUT_MS = 1000
+
+connection_parameters = {
+  size: ENV['CALL_SESSION_REDIS_POOL_SIZE'].to_i || DEFAULT_CALL_SESSION_REDIS_POOL_SIZE,
+  timeout: (ENV['CALL_SESSION_REDIS_TIMEOUT_MS'].to_i || DEFAULT_CALL_SESSION_REDIS_TIMEOUT_MS) / 1000
```

Comment:
We can just mention the seconds to be simpler no? `/ 1000` will anyways return a whole number.

Reply:
Good point.